egs2/an4/asr1/conf/deepspeed_zero2.json

```diff
@@ -0,0 +1,47 @@
+{
+    "train_batch_size": 32,
```

Comment:
`train_batch_size` (32 in this case) might be conflict with `train.yaml::batch_size` (64 in this case)